plutus-tx-constraints/src/Ledger/Tx/Constraints/OffChain.hs

```diff
@@ -277,3 +281,18 @@ lookupTxOutRef
     :: Tx.TxOutRef
     -> ReaderT (P.ScriptLookups a) (StateT P.ConstraintProcessingState (Except MkTxError)) Tx.ChainIndexTxOut
 lookupTxOutRef txo = mapReaderT (mapStateT (mapExcept (first LedgerMkTxError))) $ P.lookupTxOutRef txo
+
+lookupInlineScript
```

Comment:
Reading back the CIPs, they are actually called "reference scripts". So perhaps we can change to `lookupReferenceScript`?

Also, the name is not adequate I think. Your basically using `lookupScript` and then converting the result to a `ReferenceScript` datatype. You're not actually looking up a reference script in the lookups table.

Reply:
Ah I was thinking when outputting them the name "inline script" is better, and reserver "reference script" when they are used. But it's better to stick to the CIP naming.